frontend/public/components/utils/label-list.tsx

```diff
@@ -19,21 +22,21 @@ export const Label: React.SFC<LabelProps> = ({ kind, name, value, expand }) => {
   );
 };
 
-export class LabelList extends React.Component<LabelListProps> {
+class TranslatedLabelList extends React.Component<LabelListProps> {
```

Comment:
nit: noticed that for the components we are using `*WithTranslation` suffix.
```suggestion
class LabelListWithTranslation extends React.Component<LabelListProps> {
```

But wondering if we should rather just use the original component name as we do when use `connect`, since `withTranslation` is also just a HOC.

Would be good to align on the naming, to avoid any confusions.